test/logger.test.js

```diff
@@ -9,33 +9,36 @@ const { ecsMeta } = require('../utils');
 const chance = new Chance();
 
 describe('#setupNodeLogger()', () => {
+  let msg;
+  beforeEach(() => {
+    msg = chance.string({ symbols: false, alpha: true });
+  });
+
   describe('#json', () => {
     it('should printout the configured severity level', () => {
       const log = setupNodeLogger({ json: true, level: 'warn' });
 
-      let printout = chance.string();
+      // let msg = chance.string();
       let stdout = capture.captureStdout(() => {
-        log.warn(printout);
+        log.warn(msg);
       });
 
-      expect(stdout.indexOf(printout)).to.be.gt(-1);
+      expect(stdout.indexOf(msg)).to.be.gt(-1);
```

Comment:
`msg` isn't defined (commented out above)

Reply:
it is defined in `beforeEach`